Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -22,6 +23,7 @@ allprojects {
 apply plugin: 'com.android.application'
 
 dependencies {
+    implementation 'com.fyber:fairbid-sdk:3.15.1'
```

Comment:
I believe this one is now required (as written in the readme), but wasn't before?

Reply:
Not really - By importing the `.unitypackage` you were getting the SDKs also embedded inside. That's why the package is huge 😛 and also the reason why it includes transitive dependencies too...  a bit inconvenient.